docs/reference/enumeration-classes.txt

```diff
@@ -0,0 +1,161 @@
+===================
+Enumeration Classes
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+MongoDB\\Model\\CollectionInfo
+------------------------------
+
+Definition
+~~~~~~~~~~
+
+.. phpclass:: MongoDB\\Model\\CollectionInfo
+
+   This class models information about a collection. Instances of this class are
+   returned by traversing a :phpclass:`MongoDB\\Model\\CollectionInfoIterator`,
+   which is returned by :phpmethod:`MongoDB\\Database::listCollections()`.
+
+Methods
+~~~~~~~
+
+.. toctree::
+   :titlesonly:
+
+   /reference/method/MongoDBModelCollectionInfo-getCappedMax
+   /reference/method/MongoDBModelCollectionInfo-getCappedSize
+   /reference/method/MongoDBModelCollectionInfo-getName
+   /reference/method/MongoDBModelCollectionInfo-getOptions
+   /reference/method/MongoDBModelCollectionInfo-isCapped
+
+----
```

Comment:
We've never included horizontal lines in the docs, so I'm not sure how this might go crazy when we overhaul the CSS. 

If you're not married to them, it would be more consistent with the rest of the docs to not include the horizontal lines.